Makefile

```diff
@@ -33,15 +33,24 @@ sync-minio-version:
 	$(eval RPM = $(shell cat /tmp/downloads-minio.json | jq '.Linux."MinIO Server".amd64.RPM.download'))
 	$(eval MINIO = $(shell curl --retry 10 -Ls -o /dev/null -w "%{url_effective}" https://github.com/minio/minio/releases/latest | sed "s/https:\/\/github.com\/minio\/minio\/releases\/tag\///"))
 
-	@echo "Replacing Variables"
-
 	@cp source/default-conf.py source/conf.py
 
-	@sed -i "" "s|MINIOLATEST|${MINIO}|g" source/conf.py
-	@sed -i "" "s|DEBURL|${DEB}|g" source/conf.py
-	@sed -i "" "s|RPMURL|${RPM}|g" source/conf.py
-
-	@git add source/conf.py && git commit -m "Updating MinIO server to ${MINIO}"
+	@if [ "$(shell uname)" == "Darwin" ]; then \
```

Comment:
```suggestion
        @kname=$(uname -s)
        @case "${kname}" in \
        Darwin )
        	sed -i "" "s|MINIOLATEST|${MINIO}|g" source/conf.py; \
		sed -i "" "s|DEBURL|${DEB}|g" source/conf.py; \
		sed -i "" "s|RPMURL|${RPM}|g" source/conf.py; \
		;; \
	*) \
        	sed -i "s|MINIOLATEST|${MINIO}|g" source/conf.py; \
		sed -i "s|DEBURL|${DEB}|g" source/conf.py; \
		sed -i "s|RPMURL|${RPM}|g" source/conf.py; \
                ;; \
```

Reply:
Whoops. Munged the suggestion, will fix this. Thanks for the suggestion.